broadinstitute/pearl
# AR-76 Admin portals list

This continues porting over the admin UX from the demo, and moving things to portals from studies.  There are some controller utilities added for user login and getting user identity from the token, but I imagine that will be temporary as we move to B2C.  

TO TEST:
1. ensure your database has the ourhealth portal in it
2. start the admin UI  and admin API
3. go to localhost:3000
4. login as a valid user
5. confirm you are redirected to "localhost:3000/ourhealth", which, for now, just shows "unknown page"
6. refresh the page
7. confirm you are not logged out
